src/lib/webpack-config.js

```diff
@@ -260,6 +261,11 @@ export default env => {
 			}
 		}),
 
+		// dev-only plugins
```

Comment:
thought: maybe drop this into the [`development()` group](https://github.com/developit/preact-cli/blob/2ed133bd9c5fa92b5ee4375628fe6c67b1749775/src/lib/webpack-config.js#L326) instead?